osquery/utils/tests/aws_util_tests.cpp

```diff
@@ -64,6 +80,9 @@ TEST_F(AwsUtilTests, test_get_credentials) {
   ASSERT_EQ("FLAG_ACCESS_KEY_ID", credentials.GetAWSAccessKeyId());
   ASSERT_EQ("flag_secret_key", credentials.GetAWSSecretKey());
 
+// Profiles are not working on Windows; see the constructor of
+// OsqueryAWSCredentialsProviderChain for more information
+#if !defined(WINDOWS)
```

Comment:
If these aren't failing to compile, would you mind prefering the use of `if (isPlatform(PlatformType::TYPE_WINDOWS)) {` instead of preprocessor `#defines`?